### PR TITLE
[Fix]Update genderGap data currencies 

### PR DESCRIPTION
**Pull request by nerdlet**
## Description
Update data/ currencies csv Currency codes. 
Additional improvement of  https://github.com/CodeForAfrica/GenderGap.AFRICA/pull/48

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots

https://user-images.githubusercontent.com/3693563/151527384-11dff341-318d-4fda-9b42-58c0f87b34cd.mov

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kilemensi** [comment]
Is this related to https://github.com/CodeForAfrica/GenderGap.AFRICA/pull/48 @nerdlet ?

**Comment by kilemensi** [comment]
> > Is this related to #48 @nerdlet ?
> 
> @kilemensi Kinda. Update for #48 pushes to `gh-pages`, meaning that any change made only applies to that branch alone. Gh-pages only contains `dist` files. not project files.

That's fine @nerdlet because GitHub deploys the app from `gh-pages` only: See [settings](https://github.com/CodeForAfrica/GenderGap.AFRICA/settings/pages).

**Comment by nerdlet** [comment]
> > > Is this related to #48 @nerdlet ?
> > 
> > 
> > @kilemensi Kinda. Update for #48 pushes to `gh-pages`, meaning that any change made only applies to that branch alone. Gh-pages only contains `dist` files. not project files.
> 
> That's fine @nerdlet because GitHub deploys the app from `gh-pages` only: See [settings](https://github.com/CodeForAfrica/GenderGap.AFRICA/settings/pages).

@kilemensi True, but the `data files` and the` index.html` in the root folder will. not be up to date. So any future change made that will be pushed to dist folder will not match.

**Comment by kilemensi** [comment]
> @kilemensi True, but the `data files` and the` index.html` in the root folder will not be up to date. So any future change made that will be pushed to dist folder will not match because dist folder changes will be ahead.

I think you're supposed change data and html files in the root folder and then the `gulp` script/command will [copy](https://github.com/CodeForAfrica/GenderGap.AFRICA/blob/8a9d3c9ac599c98f0f37ae0b2467baf37bc6d5fb/gulpfile.js#L107-L110) them to `dist` folder when building the project @nerdlet

**Comment by nerdlet** [comment]
> > @kilemensi True, but the `data files` and the` index.html` in the root folder will not be up to date. So any future change made that will be pushed to dist folder will not match because dist folder changes will be ahead.
> 
> I think you're supposed change data and html files in the root folder and then the `gulp` script/command will [copy](https://github.com/CodeForAfrica/GenderGap.AFRICA/blob/8a9d3c9ac599c98f0f37ae0b2467baf37bc6d5fb/gulpfile.js#L107-L110) them to `dist` folder when building the project @nerdlet

 @kilemensi Yap, that's what I said

**Comment by nerdlet** [comment]
@kilemensi This PR is ready to be merged

**Comment by kilemensi** [comment]
> @kilemensi This PR is ready to be merged

Who said this? 😎 … Anyways, besides committing changes in `dist` folder to `master`, I guess the PR looks aaight.

File: data/currencies.csv [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
+SEN,8.90388
+SLL,0.727858
+ZAR,150.375
+SZL,8.75367
+TZS,27.5938
+TGO,15.3608
+TND,6.80858
+UGX,27.7574
+ZMW,150.375
+ZWL,16.8603
```
Comment (kilemensi) [inline data/currencies.csv:41]:
```suggestion
ZWL,16.8603

```

File: data/data.csv [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
+Senegal,SEN,2238.405854,4605.247366
+Sierra Leone,SLL,1456.625981,1956.820373
+South Africa,ZAR,10247.02276,14688.25563
+Swaziland,SZL,7655.92206,9653.67464
+Tanzania,TZS,1948.356708,2619.074651
+Togo,TGO,1340.665172,1683.369648
+Tunisia,TND,4662.455612,16787.34951
+Uganda,UGX,1778.864593,2982.206777
+Zambia,ZMW,3392.006461,3485.065249
+Zimbabwe,ZWL,2390.550252,3204.419959
```
Comment (kilemensi) [inline data/data.csv:41]:
```suggestion
Zimbabwe,ZWL,2390.550252,3204.419959

```

File: package.json [hunk 3]
```diff
@@ -43,5 +43,8 @@
     "url-search-params-polyfill": "1.3.0",
     "velocity-animate": "^1.5.0",
     "whatwg-fetch": "^3.0.0"
+  },
+  "resolutions": {
+    "graceful-fs": "^4.2.9"
```
Comment (kilemensi) [inline package.json:48]: You need this with node 10?

**Review by kilemensi — APPROVED** [review]
🚀

**Comment by nerdlet** [comment]
> > @kilemensi This PR is ready to be merged
> 
> Who said this? 😎 … Anyways, besides committing changes in `dist` folder to `master`, I guess the PR looks aaight.

@kilemensi 🤣 🤣  Okay, okay

File: package.json (hunk 3 above)
Comment (nerdlet) [reply package.json:48]: @kilemensi We need this with node v14.15.0.

**Pull request merged**